contracts/extensions/VotingReputation.sol

```diff
@@ -392,7 +400,7 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs, BasicMetaTrans
       _vote == YAY &&
       !motion.escalated &&
       motion.stakes[YAY] == requiredStake &&
-      getSig(motion.action) == CHANGE_FUNCTION_SIG &&
+      (getSig(motion.action) == SET_EXPENDITURE_STATE || getSig(motion.action) == SET_EXPENDITURE_PAYOUTS) &&
```

Comment:
Are we not missing `SET_EXPENDITURE_SLOT_PAYOUTS` as a possibility for a function that should trigger a claim delay? And if so, are we missing test(s) that should fail given it's omitted? Same for the logic where claim delays are released.